curriculum/challenges/_meta/top-learn-arrays-and-loops/meta.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "TOP Learn Arrays and Loops",
+  "isUpcomingChange": true,
+  "dashedName": "top-learn-arrays-and-loops",
+  "helpCategory": "Odin",
+  "order": 14,
+  "time": "",
+  "template": "",
+  "required": [],
```

Comment:
```suggestion
```

This will fail on main, we added some schema validation and it won't allow those empty values.